cmd/internal/cli/plugin_install_linux.go

```diff
@@ -27,10 +27,11 @@ func init() {
 //
 // singularity plugin install <path> [-n name]
 var PluginInstallCmd = &cobra.Command{
-	RunE: func(cmd *cobra.Command, args []string) error {
-		fmt.Println("Installing plugin")
-
-		return singularity.InstallPlugin(args[0], buildcfg.LIBEXECDIR)
+	Run: func(cmd *cobra.Command, args []string) {
+		err := singularity.InstallPlugin(args[0], buildcfg.LIBEXECDIR)
+		if err != nil {
+			fmt.Printf("Failed to install plugin %q: %s.", args[0], err)
```

Comment:
Let's use `sylog.Fatalf()` here instead of `fmt.Printf()`. This way we get automatic exit code handling

Reply:
Same for the rest of the `Run` functions